lib/shared/addon/utils/amazon.js

```diff
@@ -683,7 +683,7 @@ export const EKS_REGIONS = [
 ];
 
 // from https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
-export const EKS_VERSIONS = ['1.23']; // sort newest->oldest so we dont have to run any logic to sort like other provider versions
+export const EKS_VERSIONS = ['1.24', '1.23']; // sort newest->oldest so we dont have to run any logic to sort like other provider versions
```

Comment:
Until they, Amazon didn't released EKS in version `1.24`, probably this change needs to wait. Please check https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html

Reply:
It can be merged now, EKS v1.24 was released https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html